src/Backoffice/Flight/App/Resources/FlightResource.php

```diff
@@ -18,9 +16,9 @@ public function toArray(Request $request): array
     {
         return [
             'id' => $this->resource->id,
-            'airline' => AirlineResource::make($this->whenLoaded('airline')),
-            'departure_city' => CityResource::make($this->whenLoaded('departure_city')),
-            'arrival_city' => CityResource::make($this->whenLoaded('arrival_city')),
+            'airline_name' => $this->resource->airline->name,
+            'departure_city_name' => $this->resource->departureCity->name,
+            'arrival_city_name' => $this->resource->arrivalCity->name,
```

Comment:
no need for this, send the full object since you may need the rest of the information in there